src/axom/quest/DiscreteShape.hpp

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#ifndef AXOM_KLEE_DISCRETE_SHAPE_HPP
+#define AXOM_KLEE_DISCRETE_SHAPE_HPP
+
+#include <string>
+#include <vector>
+
+#include "axom/klee/Shape.hpp"
+#include "axom/mint/mesh/UnstructuredMesh.hpp"
+
+#if defined(AXOM_USE_MPI)
+  #include "mpi.h"
+#endif
+
+namespace axom
+{
+namespace quest
+{
+/*!
+  @brief Post-processed klee::Shape, with the geometry discretized and transformed
+  according to the Shape's operators.
+
+  TODO: Move this class into internal namespace.
```

Comment:
Are comments like this captured in issues so they don't get lost?

Reply:
Issue https://github.com/LLNL/axom/issues/1445